layers/ds/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+RUN pecl install ds
+RUN cp `php-config --extension-dir`/ds.so /tmp/ds.so
+
+# Build the final image from the lambci image that is close to the production environment
+FROM lambci/lambda:provided
+
+# Copy things we installed to the final image
+COPY --from=ext /tmp/ds.so /opt/bref-extra/ds.so
```

Comment:
Could you add an empty line at the end?

Reply:
Why is an empty line required? Just asking as I don't understand reasoning.

Reply:
It is a code standard in php. I’m not 100% sure why it is to be honest. 

I made my comment for consistency with other files and because github made a big warning on the diff.

Reply:
Actually, code standard for php specifies not to have an empty line at the end. All my litters will remove them. And this of course is not a php file

Reply:
Oops 'linters'

Reply:
Psr-2 : files must end with a single blank line. Ok, letter of the law for PHP. Ie. No additional lines. Trust me, it works. Suggest you look at your linters. I use easy code standard across all my projects. Actually, it's a really stupid rule unless you take it to mean, no extraneous space at the end of the file. That makes sense. If you hook easy code standard into the build chain, it will fix these automatically. 

Anyway, to play nice, if you insist I'll put a \cr in 🙂

Reply:
OK, so tried adding that extra line. Both via direct edit in Github, and via my IDE with a push. Github seems to be stripping out out again.  I think it might just be a visual thing. Does the same in master source view, although locally the extra line is there. Confirmed with gmp\Dockerfile.